CHALLENGE.md

```diff
@@ -2,22 +2,23 @@
 
 ## Prerequisites
 
-- Clone this repository to your machine
-- Select the branch (depending on the language you want to work with)
-  - The `master` branch is based on TypeScript.
-
+- Clone this repository to your machine.
+- Please, avoid changing the repository structure without a strict necessity; otherwise, motivate your choice.
+- Write the code you are usually writing. Avoid writing broken or sketchy code. We can't evaluate how good you are if the code is not good.
```

Comment:
:thinking: is this needed at all? I don't think we should warn the candidate to "please write good code 😅".

Reply:
I mentioned it explicitly because sometimes candidates say on the technical interview that they did the challenge sketchy and they usually write better code 😂

Reply:
I honestly think it should be a given of the fact that completing a challenge that gives you a ticket to the company that you want to work for should be done with reasonable code... I don't think we as a company need to say it :sweat:

Reply:
Hmm maybe it can be written that way? "We are expecting to see the best out of you for this challenge"

Reply:
This point was already discussed, we don't want people to do more than we ask.
Maybe we need better wording for it.